compiler/noirc_frontend/src/hir/comptime/value.rs

```diff
@@ -397,6 +397,29 @@ impl Value {
             value => Err(InterpreterError::CannotInlineMacro { value, location }),
         }
     }
+
+    /// Creates a value that holds an `Option``.
+    /// `option_type` must be a Type referencing the `Option` type.
+    pub(crate) fn option(option_type: Type, value: Option<Value>) -> Value {
```

Comment:
This is a bit awkward because if relies on `option_type` being an Option... I didn't know where to put this function. It could have been a stand-alone function too. Or even a method on `Type`.

Reply:
This is fine as a method I think